sklearn/multioutput.py

```diff
@@ -536,6 +536,10 @@ class labels for each estimator in the chain.
     order_ : list
         The order of labels in the classifier chain.
 
+    See also
+    --------
+    RegressorChain
```

Comment:
While you're at it, add Multi-output lassifierMixin, noting that it learns each output independently rather than chaining

Reply:
Did you mean ClassifierMixin?  And also, ClassifierMixin in ClassifierChain and RegressorMixin in RegressorChain. Am I right?

Reply:
No, phone messed it up. Multi-output lassifierMixin etc

Reply:
I think you phone messed it up again? I'm not entirely sure what you mean either ;) I thought it would be natural to add MultioutputRegressor and OneVsRestClassifier here.

Reply:
Ugh. I checked it that time! Sneaky device!

 MultioutputClassifier and MultioutputRegressor. OvR for multilabel is something I think we should consider discouraging. It's confusing to use the term ovr for binary relevance.

Reply:
Meanwhile, I googled "lassifierMixin" several times. Also git grep "lassifierMixin" showed nothing new ;) 
Hope its done now!

Reply:
A lassifierMixin is useful on a hot afternoon when you need a sweet drink.